sql/catalyst/src/main/scala/org/apache/spark/sql/internal/StaticSQLConf.scala

```diff
@@ -295,4 +295,14 @@ object StaticSQLConf {
       .version("3.1.0")
       .stringConf
       .createWithDefault("")
+
+  val DATA_FRAME_DEBUGGING_ENABLED =
+    buildStaticConf("spark.sql.dataFrameDebugging.enabled")
```

Comment:
Let's name sth like `spark.python.sql.dataFrameDebugging.enabled` so it can be specific to python